domains/robloxbot.json

```
@@ -0,0 +1,15 @@
+{
+  "description": "Robloxbot Inc",
+  "repo": "https://github.com/Gatshja/register",
+  "owner": {
+    "username": "Gatshja",
+    "email": "robloxbotthh@gmail.com"
+{
+  "record": {
+    "NS": [
+      "celine.ns.cloudflare.com", (reason: Subdomain and Project Web)
```

Adding that line will make the file invalid.
```suggestion
      "celine.ns.cloudflare.com",
```

---

Take minutes fixing...

---

Done you can review.